featurebyte/query_graph/sql/aggregator/window.py

```diff
@@ -487,13 +495,19 @@ def construct_aggregation_sql(  # pylint: disable=too-many-arguments
         for serving_name in serving_names:
             group_by_keys.append(quoted_identifier(serving_name))
 
-        if value_by is None:
-            inner_agg_result_names = agg_result_names
-            inner_group_by_keys = group_by_keys
-        else:
-            inner_agg_result_names = [
-                f"inner_{agg_result_name}" for agg_result_name in agg_result_names
-            ]
+        inner_agg_result_names = agg_result_names
+        inner_group_by_keys = group_by_keys
+        if value_by is not None:
```

Comment:
down the line, it would nice to consolidate the `value_by` logic here with some logic that looks similar in the `get_groupby_expr` in the `groupby_helper`.